polymer.json

```diff
@@ -7,5 +7,14 @@
     "src/shop-cart.html",
     "src/shop-checkout.html",
     "src/lazy-resources.html"
+  ],
+  "sourceGlobs": [
+   "src/**/*",
+   "images/**/*",
+   "bower.json"
+  ],
+  "includeDependencies": [
+    "manifest.json",
+    "bower_components/webcomponentsjs/webcomponents-lite.min.js"
```

Comment:
Great question! We're actually still figuring that out as we go. If you're using the 'polymer-build' library yourself you can apply different optimizations/transforms to source files and dependency files. But today in the CLI we apply the same optimizations to both, so there's no real difference if you're using 'polymer build' on the command line.

We've seen this question a few times now, we should spend some time clarifying this in the README.